src/main/resources/2.7.x.conf

```diff
@@ -118,7 +121,7 @@ templatecontrol {
 
           "com.h2database" = """    play "com.h2database:h2:1.4.197""""
 
-          "scalatestplus-play" = """    playTest "org.scalatestplus.play:scalatestplus-play_\$scalaVersion:4.0.0-RC2""""
+          "scalatestplus-play" = """    playTest "org.scalatestplus.play:scalatestplus-play_$scalaVersion:4.0.0-RC2""""
```

Comment:
Is this right?

Reply:
yes, this is right for now. We have a few samples using scalatest plus. They will all get broken if we escape the $. 

When we don't escape the $, we will only break play-scala-seed.g8. So, better to break only one and fix it by hand. 

And a fix is coming, see #102

Reply:
I see.  Thanks for summarising the state!